src/main/java/ua/mykolamurza/chatullo/Command/UnmuteCommand.java

```diff
@@ -0,0 +1,58 @@
+package ua.mykolamurza.chatullo.Command;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.plugin.java.JavaPlugin;
+
+import java.io.*;
+
+public class UnmuteCommand implements CommandExecutor {
+
+    private final JavaPlugin plugin;
+
+    public UnmuteCommand(JavaPlugin plugin) {
+        this.plugin = plugin;
+    }
+
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if (args.length == 0) {
+            sender.sendMessage("Usage: /unmute <player>");
+            return true;
+        }
+
+        String targetPlayerName = args[0];
+
+        if (removeMutedPlayerFromFile(targetPlayerName)) {
+            sender.sendMessage("Player " + targetPlayerName + " has been unmuted.");
+        } else {
+            sender.sendMessage("Player " + targetPlayerName + " is not muted.");
+        }
+
+        return true;
+    }
+
+    private boolean removeMutedPlayerFromFile(String playerName) {
+        File file = new File(plugin.getDataFolder(), "muted_players.txt");
+        File tempFile = new File(plugin.getDataFolder(), "temp_muted_players.txt");
```

Comment:
Hard solution. Could be something like this [code below]. 
I don't want to make a full copy of the file without one line. 
Honestly, your solution is excellent too, and yes, I agree with you that the best idea there is SQLite or MySQL entity.

```java
try {
    File inputFile = new File(fileName);
    BufferedReader reader = new BufferedReader(new FileReader(inputFile));

    String currentLine;
    List <String> lines = new ArrayList <> ();
    while ((currentLine = reader.readLine()) != null) {
        String trimmedLine = currentLine.trim();
        if (!trimmedLine.equals(playerName)) {
            lines.add(currentLine);
        }
    }
    reader.close();

    BufferedWriter writer = new BufferedWriter(new FileWriter(inputFile));
    for (String line: lines) {
        writer.write(line);
        writer.newLine();
    }
    
    writer.close();
} catch (IOException ex) {
    ex.printStackTrace();
}
```

Reply:
You could resolve this message without changes. It is not necessary.